File: SoftUni Lessons/Python Development/Python Basics April 2019/Past Exams and Problems/8 - 6 and 7 April 2019/02. Godzilla vs. Kong.py
budget = float(input())
people_count = int(input())
one_dress = float(input())

decor = budget * 0.1
dresses = people_count * one_dress

if people_count > 150:
    dresses -= dresses * 0.1

total = dresses + decor

if total > budget:
    print("Not enough money!")
    print(f"Wingard needs {abs(total - budget):.2f} leva more.")
else:
    print("Action!")
    print(f"Wingard starts filming with {abs(budget - total):.2f} leva left.")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/21. МИНАЛИ ИЗПИТИ/05. Online Exam - 6 and 7 April 2019/02. Godzilla vs. Kong.js
function godzillaVsKong(params) {
    let budget = Number(params.shift());
    let people = Number(params.shift());
    let priceForOne = Number(params.shift());

    let decor = budget * 0.1;
    
    if (people > 150) {
        priceForOne *= 0.9
    }

    let price = (people * priceForOne) + decor;
    let difference = Math.abs(budget - price);

    if (price > budget) {
        console.log(`Not enough money!`);
        console.log(`Wingard needs ${difference.toFixed(2)} leva more.`);
    } else {
        console.log(`Action!`);
        console.log(`Wingard starts filming with ${difference.toFixed(2)} leva left.`);
    }
}


godzillaVsKong(['20000', '120', '55.5']);  // Expected Output:
// Action!
// Wingard starts filming with 11340.00 leva left.

godzillaVsKong(['15437.62', '186', '57.99']);  // Expected Output:
// Action!
// Wingard starts filming with 4186.33 leva left.

godzillaVsKong(['9587.88', '222', '55.68']);  // Expected Output:
// Not enough money!
// Wingard needs 2495.77 leva more.